docs/hello_nextflow/02_hello_channels.md

```diff
@@ -0,0 +1,880 @@
+
+!!! note
+
+    It is technically possible to achieve the same results by using a different channel factory, [`Channel.fromList`](https://nextflow.io/docs/latest/reference/channel.html#fromlist), which includes an implicit mapping step in its operation.
+    Here we chose not to use that in order to demonstrate the use of an operator on a fairly simple use case.
+
+#### 3.2.1. Add the `flatten()` operator
+
+To apply the `flatten()` operator to our input channel, we append it to the channel factory declaration.
+
+In the workflow block, make the following code change:
+
+_Before:_
+
+```groovy title="hello-channels.nf" linenums="31"
+    // create a channel for inputs
+    greeting_ch = Channel.of(greetings_array)
+```
+
+_After:_
+
+```groovy title="hello-channels.nf" linenums="31"
+    // create a channel for inputs
+    greeting_ch = Channel.of(greetings_array)
+                         .flatten()
+```
+
+Here we added the operator on the next line for readability, but you can add operators on the same line as the channel factory if you prefer, like this: `greeting_ch = Channel.of(greetings_array).flatten()`
+
+#### 3.2.2. Add `view()` to inspect channel contents
+
+We could run this right away to test if it works, but while we're at it, we're also going to add a couple of [`view()`](https://www.nextflow.io/docs/latest/reference/operator.html#view) directives, which allow us to inspect the contents of a channel.
+You can think of `view()` as a debugging tool, like a `print()` statement in Python, or its equivalent in other languages.
+
+In the workflow block, make the following code change:
+
+_Before:_
+
+```groovy title="hello-channels.nf" linenums="31"
+    // create a channel for inputs
+    greeting_ch = Channel.of(greetings_array)
+                         .flatten()
+```
+
+_After:_
+
+```groovy title="hello-channels.nf" linenums="31"
+    // create a channel for inputs
+    greeting_ch = Channel.of(greetings_array)
+                         .view { "Before flatten: $it" }
+                         .flatten()
+                         .view { "After flatten: $it" }
+```
+
+Here `$it` is an implicit variable that represents each individual item loaded in a channel.
+
+#### 3.2.3. Run the workflow
+
+Finally, you can try running the workflow again!
+
+```bash
+nextflow run hello-channels.nf
+```
+
+This time it works AND gives us the additional insight into what the contents of the channel look like before and after we run the `flatten()` operator:
+
+```console title="Output" linenums="1"
+ N E X T F L O W   ~  version 24.10.0
+
+Launching `hello-channels.nf` [tiny_elion] DSL2 - revision: 1d834f23d2
+
+executor >  local (3)
+[8e/bb08f3] sayHello (2) | 3 of 3 ✔
+Before flatten: [Hello, Bonjour, Holà]
+After flatten: Hello
+After flatten: Bonjour
+After flatten: Holà
+```
+
+You see that we get a single `Before flatten:` statement because at that point the channel contains one item, the original array.
```

Comment:
So far, I have seen the words value, element, and item referring to the same thing (what is known as a channel element) in part II. I'd rather stick to elements (and items being the individual entries within an element).

Reply:
Ok I agree with the push for consistency. But it's okay if some instances are left for later; getting the reorg merged soon is more important than having every word be perfect.